woocommerce/woocommerce-gutenberg-products-block
# Fix Markdown table in payment-method-integration.md

### Description
Fix the incorrect formatting in the [Payment Method Integration for the Checkout Block](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/docs/third-party-developers/extensibility/checkout-payment-methods/payment-method-integration.md) - table is not rendering correctly in the [Props Fed to Payment Method Nodes](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/docs/third-party-developers/extensibility/checkout-payment-methods/payment-method-integration.md#props-fed-to-payment-method-nodes) section

This PR contains whitespace changes only.

### Screenshots
| Before | After |
| ------ | ----- |
|    <img width="1076" alt="image" src="https://user-images.githubusercontent.com/20098064/212272328-1d4f8e33-d5d7-476f-b1a4-8b9ceda603f9.png">   |.   <img width="1084" alt="image" src="https://user-images.githubusercontent.com/20098064/212272348-313a8cb6-bda1-4733-9513-833ae11fe9fd.png"> | 

## Testing
### Testing steps
- Before: check the table on `trunk`: [before](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/docs/third-party-developers/extensibility/checkout-payment-methods/payment-method-integration.md#props-fed-to-payment-method-nodes)
- After: check the table in this PR: [after](https://github.com/woocommerce/woocommerce-blocks/blob/7e39f49632742d1b88a9f390219433217b775158/docs/third-party-developers/extensibility/checkout-payment-methods/payment-method-integration.md#props-fed-to-payment-method-nodes) 😍 


### WooCommerce Visibility
